app/javascript/packages/document-capture/components/form-steps.jsx

```diff
@@ -79,24 +78,44 @@ export function getStepIndexByName(steps, name) {
   return steps.findIndex((step) => step.name === name);
 }
 
+/**
+ * Returns the first element matched to a field from a set of errors, if exists.
+ *
+ * @param {FormStepError<Record<string,Error>>[]} errors Active form step errors.
+ * @param {Record<string,FieldsRefEntry>} fields Current fields.
+ *
+ * @return {HTMLElement|null|undefined}
+ */
+function getFieldActiveErrorFieldElement(errors, fields) {
+  const error = errors.find(({ field }) => !!fields[field]?.element);
```

Comment:
the `!!` is not needed right? Since `find` will take normal truthiness from its callback anyways?

Reply:
> the `!!` is not needed right? Since `find` will take normal truthiness from its callback anyways?

Right, not strictly necessary. I guess I'd imagine there could be some edge cases, though I can't imagine any edge cases which still wouldn't be edge cases with casting the value to a boolean. 🤷 

I'll simplify it.

Reply:
> Right, not strictly necessary. I guess I had imagined there could be some edge cases, though I can't imagine any edge cases which still wouldn't be edge cases with casting the value to a boolean. 🤷
> 
> I'll simplify it.

Updated in 183e1ce.